pxr/base/plug/testPlug.py

```diff
@@ -275,10 +275,12 @@ def test_ErrorCases(self):
         self.assertEqual(listener.numReceived, 0)
 
         # try to load an unloadable plugin
-        badPlugin = Plug.Registry().GetPluginForType('TestPlugUnloadable')
-        self.assertIsNotNone(badPlugin)
-        with self.assertRaises(RuntimeError):
-            badPlugin.Load()
+        # NOTE: This test is broken on Windows ARM64, so is switched off
+        if not "ARM" in os.environ.get('PROCESSOR_IDENTIFIER'):
```

Comment:
```suggestion
        if "ARMv" not in os.environ.get('PROCESSOR_IDENTIFIER', ''):
```

Matches `ARMv` check in `build_usd.py` above. Also needed to specify a default value to `get` to avoid errors on other platforms.